src/Tribe/Google_Data_Markup/Event.php

```diff
@@ -23,7 +23,28 @@ protected function build_data() {
 			$event_data[ $id ]->location          = new stdClass();
 			$event_data[ $id ]->location->{'@type'} = 'Place';
 			$event_data[ $id ]->location->name    = tribe_get_venue( $post->ID );
-			$event_data[ $id ]->location->address = strip_tags( str_replace( "\n", '', tribe_get_full_address( $post->ID ) ) );
+			$event_data[ $id ]->location->address = new stdClass();
+
+			if ( tribe_get_address( $post->ID ) ) {
+				$event_data[ $id ]->location->address->streetAddress    = tribe_get_address();
+			}
+
+			if ( tribe_get_city( $post->ID ) ) {
+				$event_data[ $id ]->location->address->addressLocality    = tribe_get_city();
```

Comment:
Why the odd spacing around the `=`? Should be closed to one space either side of it.